qri/dataset.py

```diff
@@ -54,6 +54,29 @@ def __init__(self, obj):
     def __repr__(self):
         r = build_repr(self)
         return 'Structure(%s)' % r
+    
+    def _repr_html_(self):
```

Comment:
I was thinking that if you like the option of seeing the schema with or without the other structure fields (checksum, etc), maybe we could make `Schema` into its own type with its own `_repr_html_`, so you could do `ds.structure.schema` or `ds.structure` depending on which you want.